docs/general/researching-and-asking-for-help.md

```diff
@@ -0,0 +1,39 @@
+As a Hacker, learning things about your target is the first step in attacking it. Besides the knowledge you need to learn on how to utilise the tools that you will be using, researching information gathered from your target is an essential step in knowing **how** to attack your target.
+
+**[MuirlandOracle](https://tryhackme.com/p/MuirlandOracle)** has created a fantastic room on the various things that are worth researching - both for attacking a target and learning how to use the tools to do so. I greatly encourage **[participating in the room](https://tryhackme.com/room/introtoresearch)**, to gain these vital skills.
+
+
+## Asking for Help
+Whilst TryHackMe is a platform aimed at introducing Cybersecurity to beginners, we assume that there is a willingness to learn, and don't like to spoon-feed solutions. In the long-run - there is no beneficiary to you!
+
+We **really** encourage platform users to try and solve problems they come across within rooms themselves - patience is key - like in the real world. Sometimes taking a "breather"; A couple of minutes away from the task and re-approaching it with a fresh mindset will solve your issue alone.
+
+However, even after you've extensively researched and you still need a point in the right direction, that's what the community is here for - and arguably what makes it so great.
+
+![dont just ask to ask](https://i.imgur.com/WBmLS1c.png)
+
+When asking for help, don't just ask! I.e. "can I have help?" Instead, state your problem and the steps you have performed to try and resolve it. For example, a great "ask" is:
+
+> Hi, I'm having an issue with Task #43 - Question #1 in the room [Learn Linux](https://tryhackme.com/room/zthlinux). I have tried at looking **x**,**y**,**z** and I have googled what I think might be a vulnerability. However, I can't seem to find any way to progress. Can someone offer some advice? Thanks!
+
+Greatly formatted questions are both really helpful in exchanges between users. The Discord can be slow at times, so formatting your question similar to the one above skips the process of trying to figure out where you're stuck and on what room.
+
+### Mentors
+A fantastic attribute to TryHackMe is the community itself. Other users like to help each other out - and it's wonderful. We really encourage users to continue helping each other out.
+
+However, we can't solely rely on users to be there 24/7 for each other, nor validate what they are saying. TryHackMe has users who throughout time have consistently guided other users through issues they experience...from technical issues relating to connecting to the network to providing resources to help resolve challenges.
+
+**These Mentors, much like Moderators are not paid** and are users of the TryHackMe platform such as yourself! Because of this, please be patient. If your answer isn't answered by any user - someone will get too it as and when they can! There is no need to post your issue in every and all channels.
```

Comment:
Only one issue I noticed here is that `"too"` should be `"to"` in the sentence `"someone will get too it as and when they can!"`

Reply:
Thanks for noticing that! Apologies for the oversight :)

Reply:
Did you want to put in the change? @CMNatic

Reply:
Sure, I'll change and open as a new PR @Kent55

Reply:
> Thanks for noticing that! Apologies for the oversight :)
> 
> Shall I fix typo and open as new PR? Or are you able to resolve and merge directly?

^^^ yes please :)